File: src/Applications/SimpleApi/Model/Utils/Result/ResponseData.cs


namespace Model.Utils.Result
{
    /// <summary>
    /// 接口输出数据
    /// </summary>
    public class ResponseData
    {
        /// <summary>
        /// 操作结果
        /// </summary>
        public bool Success { get; set; }

        /// <summary>
        /// 错误代码
        /// <see cref="Model.Utils.Result.ErrorCode"/>
        /// </summary>
        public int ErrorCode { get; set; }

        /// <summary>
        /// 消息
        /// </summary>
        public string Message { get; set; }
    }
}

File: src/Applications/SimpleApi/Api/wwwroot/plug-in/naiveupload/Model/ResponseData.ts
/**
 * 接口输出数据
 * */
class ResponseData {
    /**
     * 操作结果
     */
    Success: boolean;

    /**
     *错误代码
     */
    ErrorCode: number;

    /**
     *消息
     */
    Message: string;
}

/**
 * 接口输出数据
 * */
class ResponseData_T<T> extends ResponseData {
    /**
     * 数据
     */
    Data: T;
}